tests/test_app_tts_output/ttstest.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Yeah, Caterpillar edits this file, since it's an HTML file. Given there's been bugs with HTML processing before, I'd prefer an overcomplicated example than a bare minimum one to make sure that nothing strange happens (like adding extra HTML tags and moving the DOCTYPE around, as has happened previously).